bullet-featherstone/src/SDFFeatures.cc

```diff
@@ -639,9 +639,14 @@ bool SDFFeatures::AddSdfCollision(
     return false;
   }
 
-  double mu = 1.0;
-  double mu2 = 1.0;
-  double restitution = 0.0;
+//  double mu = 1.0;
```

Comment:
remove?

Reply:
Right, I was experimenting with the default parameters to see how they impact the collision behavior. I've returned them back to their default values here [862411b](https://github.com/gazebosim/gz-physics/pull/387/commits/862411b02426503ef149ecdd3ff168fe56360435)